src/device-registry/utils/create-location.js

```diff
@@ -13,7 +13,12 @@ const log4js = require("log4js");
 const logger = log4js.getLogger(
   `${constants.ENVIRONMENT} -- create-location-util`
 );
-const { kafkaProducer } = require("../config/kafkajs");
+
+const { Kafka } = require("kafkajs");
```

Comment:
Addressing a runtime for Kafka producer where things were undefined.
So brought all the configurations into each respective util to prevent this cyclic dependency runtime error.

Repetition of code but prevent this bug from happening ever again, hopefully!

@BenjaminSsempala @NoahNsimbe